changelog/v1.19.0-patch4/aws-sts-paranoia.yaml

```diff
@@ -0,0 +1,8 @@
+changelog:
+- type: FIX
+  issueLink: https://github.com/solo-io/gloo/issues/5253
+  resolvesIssue: false
+  description: >
+    Add new sts refresh timer in addition to filewatch.
```

Comment:
won't this still mean up to 14 minutes of failed requests in the event we miss an event on the filewatch? seems worth documenting and making timeout configurable, if so